### ISSUE TITLE
Can security rules provide a global default option?  I have more than 100 tables. I don't want to write 100 rules.  ...

### ISSUE THREAD

**Issue by hyingreborn** [spaceuptech/space-cloud#167]
like this:
```
collections: 
  default:
    isRealtimeEnabled: false
    rules:
      create:
              rule: allow
            read:
              rule: allow
            update:
              rule: allow
            delete:
              rule: allow
  todos:
    isRealtimeEnabled: true
    rules:
      create:
              rule: authenticated
            read:
              rule: authenticated
            update:
              rule: allow
            delete:
              rule: allow
  ...
```

**Comment by YourTechBud** [comment]
That's an amazing suggestion. @Jayesh333 lets get this done.

**Comment by YourTechBud** [comment]
@hyingreborn, I have added this feature in branch `v0.7.0`. It should be merged in master (and hence made live) within a couple of days. Thanks for the awesome idea.

**Comment by hyingreborn** [comment]
That's great!

**Comment by hyingreborn** [comment]
@YourTechBud 
I thought for a moment, should the security rules feature be moved to the user management side? For the global, you should be able to access all the tables, but for different users, you need to set the access rules. What do you think?

**Comment by YourTechBud** [comment]
@hyingreborn thats actually a good idea. #126 talks moving the entire config to a table / collection. Why don't you join out discord server to help me understand this better? Here's a link: https://discord.gg/ypXEEBr

Also, the `default` collection rule is implemented now and is live in `v0.7.0`. Thanks for the suggestion!
